eth/utils/state.py

```diff
@@ -1,14 +1,50 @@
+from typing import (
+    Dict,
+    Iterable,
+    Tuple,
+    Union,
+)
+
+from mypy_extensions import (
+    TypedDict,
+)
+
 from eth_utils import (
     to_tuple,
 )
 
+from eth_typing import (
```

Comment:
nitpick: `eth_typing` should be above `eth_utils` (alphabetical order within 3rd party libraries)